tests/helpers.py

```diff
@@ -1,6 +1,5 @@
 import pickle
 
-from tests.quirks import KNOWN_GYM_CLOSE_BROKEN
```

Comment:
I think this will leave dead code? Best to remove `KNOWN_GYM_CLOSE_BROKEN` if it's now unused.

Reply:
I think I already removed?

Reply:
@gonzaiva Can you confirm that we are not going to use `KNOWN_GYM_CLOSE_BROKEN` anywhere now and in the future.

Reply:
No need to preemptively retain dead code. If it has no references, it is dead and should be removed. If it's needed in the future, it can be revived from the commit history.

Reply:
OK i got it.